### DISCUSSION TITLE
OVOS dated today: the blebop after I say Hey Mcroft

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#224] [Q&A] [answered]
OVOS dated today: the blebop after I say Hey Mcroft - that volume is very low.  the answer the MC speaks is at a high volume (I reviews ALSAMIXER and all it goofd.

what dictates the volume for the  bleep after I say HEY MC ?

**Comment by goldyfruit** [comment]
Acknowledge sound is a `.wav` file handled by default by `aplay` (ALSA) but depending your TTS, sometime the generated files will be `.mp3` which might be handled by `paplay` (PulseAudio) or `pw-play` (PipeWire) which could have different sound levels depending the hardware.

Got this behaviour with a Google AIY Voice Kit V1.

**Comment by geetee24** [comment]
Can you tell me the paths for the source file and the destination file and
then how would I set that volume using any of the methods that may be an
action? I am consuming the standard OVOS raspberry pie image, so I don’t
know what my actual speaking engine is. How do I find out what that may be?

 Thank you.

**Comment by goldyfruit** [comment, marked as answer]
Default TTS will be Piper provided by the OVOS TTS public servers: https://github.com/OpenVoiceOS/status

I think on RaspOVOS, the path to the file is `~/.venvs/ovos/lib/python3.11/site-packages/ovos_audio/res/snd/start_listening.wav`.

RaspOVOS still use PulseAudio so you will have to set `play_wav_cmdline` and `play_mp3_cmdline` in your `mycroft.conf` to `paplay` with the `--volume` argument _(0 = mute, 65536 = 100%)_

https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf#L59-L64

```json
  "play_wav_cmdline": "paplay --volume 65536 %1 ",
  "play_mp3_cmdline": "paplay --volume 65536 %1",
```

This will set the volume as 100%.

**Reply by geetee24** [reply]
this doesnt work.  the root cause is the volume of the actual audio file was recorded way too low.

**Reply by JarbasAl** [reply]
the actual file is the same from mycroft

if you are so sure of the root cause, guess that means this is a mycroft bug then, must be happening in your picrofts too

**Reply by geetee24** [reply]
i find that if i raise the volume 
 "play_wav_cmdline": "paplay --volume 65536 %1 ",
  "play_mp3_cmdline": "paplay --volume 65536 %1",

it makes the bleep bleep perfect BUT then OVOS's voice is hugely loud.
so what is going on?

**Reply by goldyfruit** [reply]
Reduce the value of `"play_mp3_cmdline": "paplay --volume 65536 %1",`

**Reply by geetee24** [reply]
nope.  it affects globally.   ovos has 2 volumes.  1 for everything. 1 for just this file

**Reply by geetee24** [reply]
the issue is on ovos what is controlling the volume that mpg123 uses?  i cant change alsamixer for that is a global change.

**Reply by JarbasAl** [reply]
OVOS does not control that volume at all, it just tells the OS to play the sounds, I'm not sure of the root cause of your issue and why they would be different

the suggestions from @goldyfruit is the only way to force OVOS to indirectly control the volume, otherwise it's all up to the operating system

**Reply by geetee24** [reply]
thanks. for trying.  can you tell me what interface (pulse, etc) ovos is using?  I am trying to also see what mpg123 is using

**Reply by JarbasAl** [reply]
OVOs will use whatever your OS uses, it could be pulseaudio, pipewire or in ancient setups just pure alsa

OVOS only uses the playback commands and then let's the OS do it's usual thing without requiring any specific audio interface

**Comment by geetee24** [comment]
interesting fact: mpg123 has the low volume issue.  aplay is same volume as
ovos :)

**Reply by goldyfruit** [reply]
`aplay` uses ALSA layer directly and bypasses PipeWire or PulseAudio. `aplay` can't play mp3 files.
